.github/workflows/ci-develop.yaml

```diff
@@ -178,12 +178,29 @@ jobs:
           R_IMAGE_NAME: ${{ format('{0}/{1}:{2}-r-ci', steps.login-ecr.outputs.registry, needs.build-docker.outputs.repo-name, needs.build-docker.outputs.tag) }}
 
       - id: test
-        name: Run unit tests
+        name: Test and Generate Code Coverage Report
         run: |-
-          docker run -v $(pwd):/python:rw --env CLUSTER_ENV=test --net="host" --entrypoint /usr/bin/env $PYTHON_IMAGE_NAME python3 -m pytest .
+          pip install pytest-cov
+          docker run -v $(pwd):/python:rw --env CLUSTER_ENV=test --net="host" --entrypoint /usr/bin/env $PYTHON_IMAGE_NAME python3 -m pytest --cov=./ .
         env:
           PYTHON_IMAGE_NAME: ${{ format('{0}/{1}:{2}-python-ci', steps.login-ecr.outputs.registry, needs.build-docker.outputs.repo-name, needs.build-docker.outputs.tag) }}
 
+      - id: do-ls-docker
+        name: DEBUG do an ls in docker
+        run: |-
+          pip install pytest-cov
+          docker run -v $(pwd):/python:rw --env CLUSTER_ENV=test --net="host" --entrypoint /usr/bin/env $PYTHON_IMAGE_NAME ls
+        env:
+          PYTHON_IMAGE_NAME: ${{ format('{0}/{1}:{2}-python-ci', steps.login-ecr.outputs.registry, needs.build-docker.outputs.repo-name, needs.build-docker.outputs.tag) }}
+
+      - name: DEBUG See output
+        run: |-
+          ls ./src
```

Comment:
this probably needs to go

Reply:
Yes, this is part of the debugging I was doing, the code coverage action is run correctly but then I don't know how to make the action codecov/codecov-action@v1 run in the docker container

Reply:
although currently I am trying the alternative (copy the resulting code coverage file to local)